lib/modules/idea-single-widgets/index.js

```diff
@@ -28,15 +28,40 @@ const fields = [
       choices: [
           {
               value: true,
-              label: "Yes"
+              label: "Yes",
+              showFields: ['shareChannelsSelection']
           },
           {
               value: false,
               label: "No"
           },
-      ],
-      def: true

```

Comment:
Waarom is deze default weggehaald? Is dit niet de standaard functionaliteit. Als we deze weghalen dan zouden de Amsterdam sites dus zonder share buttons komen te zitten?